src/isaw.facultycv/isaw/facultycv/content/profile.py

```diff
@@ -200,10 +200,12 @@ def profile_links(self):
         results = []
         fullname = self.Title()
         for link in links:
+            parsed = urlparse(link)
```

Comment:
I feel like we need a try/except here in case of really strange entries, unless all possible strings are valid URLs and won't ever throw an exception?

Reply:
I was not able to break this with my attempts:

```
>>> from urlparse import urlparse
>>> urlparse("")
ParseResult(scheme='', netloc='', path='', params='', query='', fragment='')
>>> urlparse("Foo and bar - Ünicode chars - /// random slashes")
ParseResult(scheme='', netloc='', path='Foo and bar - \xc3\x9cnicode chars - /// random slashes', params='', query='', fragment='')
>>> 
```

I wouldn't know what exception to catch here, so I'd leave it as is.